src/usr/local/www/services_dhcp.php

```diff
@@ -764,7 +768,12 @@ function build_pooltable() {
 }
 
 if ($tabscounter == 0) {
-	print_info_box(gettext("The DHCP Server can only be enabled on interfaces configured with a static IPv4 address. This system has none."));
+	if ($have_small_subnet) {
+		$sentence2 = gettext("This system's static IPv4 interfaces have subnet 31 or 32.");
```

Comment:
Maybe something like this instead:
`sprintf(gettext("The static IPv4 subnet configured on this interface has a CIDR mask of %s, which does not contain enough addresses."), htmlspecialchars($oc['subnet']));`